lib/brains/tests/samples/customer.py

```diff
@@ -0,0 +1,257 @@
+minimal = """
+<?xml version="1.0" encoding="UTF-8"?>
+<customer>
+  <id>minimal</id>
+  <merchant-id>...</merchant-id>
+  <first-name>Charity</first-name>
+  <last-name>Smith</last-name>
+  <company nil="true"/>
+  <email nil="true"/>
+  <phone nil="true"/>
+  <fax nil="true"/>
+  <website nil="true"/>
+  <created-at type="datetime">2015-04-03T15:20:04Z</created-at>
+  <updated-at type="datetime">2015-04-03T15:20:04Z</updated-at>
+  <custom-fields/>
+  <credit-cards type="array"/>
+  <addresses type="array"/>
+</customer>
+"""
```

Comment:
This python module is going to get pretty wild. I suggest making them into xml files like ``samples/customers/minimal.xml``, ``samples/customers/full.xml``, etc, and just having ``http_do()`` load the XML files.